CMakeLists.txt

```diff
@@ -136,12 +136,6 @@ dd4hep_configure_output( OUTPUT "${PROJECT_BINARY_DIR}" INSTALL "${CMAKE_INSTALL
 # Configure ROOT
 find_package (ROOT 6.08 REQUIRED CONFIG)
 DD4HEP_SETUP_ROOT_TARGETS()
-# ROOT no longer finds nlohmann_json unless we ask for the ROOTEve component, so we ask
-# for the ROOTEve component if the ROOTEve component exists.
-# DDEve and one executable in UtilityApps depend on this
-if(TARGET ROOT::ROOTEve)
```

Comment:
Please just leave this here.

In your build of ROOT there shouldn't be any ROOT::ROOTEve, so it won't hurt your use case.